tests/mxnet/test_hook_collections.py

```diff
@@ -0,0 +1,61 @@
+# Standard Library
+import datetime
```

Comment:
why added in this PR? Doesn't look like these are related.

Reply:
https://github.com/awslabs/sagemaker-debugger/pull/150#discussion_r389860200